packages/code-du-travail-frontend/pages/modeles-de-courriers/[slug].jsx

```diff
@@ -61,10 +62,13 @@ class ModeleCourrier extends React.Component {
     const [filename] = fileUrl.match(/[^/]+$/);
     const [, extension] = filename.split(/\.([a-z]{2,4})$/);
     const filesizeFormated = Math.round((filesize / 1000) * 100) / 100;
+    const category = `Modèle ${
+      type === "fichier" ? "à télécharger" : `de ${type}`
```

Comment:
Faudrait juste voir si y'a pas de `type`, comment on le gère

Reply:
A priori on controle cette donnée vu qu'elle est issue du json venant de cdtn-docx-to-html.
Quand on le stockera dans la DB, on pourra le mettre en champs obligatoire pour sécuriser mais normalement le cas null ne devrait pas se pésenter